pkg/certs/util.go

```diff
@@ -418,3 +418,51 @@ func GenerateCertificateRenewRequestSameKey(cert *tls.Certificate) (retCsr []byt
 	retCsr = EncodeCertRequestPEM(x509CertReq)
 	return
 }
+
+func GenerateExpiredClientCertificate(name string) (*x509.Certificate, *rsa.PrivateKey, error) {
```

Comment:
This is not a util functon we should be including . This is a test function

Reply:
I moved this function to cert_utils.go